import_3dm/converters/curve.py

```diff
@@ -149,7 +149,10 @@ def import_polycurve(rcurve, bcurve, scale):
 
 CONVERT[r3d.PolyCurve] = import_polycurve
 
-def import_curve(og,context, n, Name, Id, layer, rhinomat, scale):
+def import_curve(ob,context, n, layer, rhinomat, scale, option):
```

Comment:
This should be left as importing the object geometry (`ob.Geometry`) to be consistent with the other converters.

The `Name` and `Id` values below are not used anywhere... ?

Reply:
Ok, I see. So we pass the whole object in and extract the attributes in the conversion functions. Doesn't appear to be a problem :)